Tests/ResilientDecodingTests/ResilientArrayTests.swift

```diff
@@ -73,9 +81,11 @@ final class ResilientArrayTests: XCTestCase {
       """,
```

Comment:
Let's fix the method name `testResilientlyDecodesArratWithInvalidElements()`. "Arrat" -> "Array"